include/CLI/App.hpp

```diff
@@ -2761,7 +2761,12 @@ class App {
         }
         int min_num = (std::min)(op->get_type_size_min(), op->get_items_expected_min());
         int max_num = op->get_items_expected_max();
-
+        // check container like options to limit the argument size to a single type if the allow_extra_flags argument is
+        // set.
+        if(max_num >= detail::expected_max_vector_size / 16 && !op->get_allow_extra_args()) {
```

Comment:
Why 16?

Reply:
No very good reason.  I realized if I was going to make this < it had to be at least /4 since I use the max_size/2 in another spot and figured it should be a multiple of 2.  So 4/8/16/32/64 probable doesn't make any difference right now.